src/crypto/wolfssl/cryptography_interface_wolfssl.template.c

```diff
@@ -422,6 +422,20 @@ static int32_t cryptography_encrypt(uint8_t* data_out, size_t len_data_out,
             }
             break;
 
+        // TODO: Confirm same process as above for SIV
```

Comment:
Should these three new comment blocks be removed?

Reply:
Yes, good catch.